### ISSUE TITLE
Parse method appending previous prediction results

### ISSUE THREAD

**Issue by srijiths** [golastmile/rasa_nlu#346]
I deployed a sentiment model and below are the inputs and predictions respectively
```
1. Cognizant shares went up

{
    "entities": [
        {
            "end": 9,
            "entity": "Company",
            "extractor": "ner_crf",
            "processors": [
                "ner_synonyms"
            ],
            "start": 0,
            "value": "Cognizant Technology Solutions"
        },
        {
            "end": 16,
            "entity": "feature",
            "extractor": "ner_crf",
            "start": 10,
            "value": "shares"
        },
        {
            "end": 24,
            "entity": "positive",
            "extractor": "ner_crf",
            "start": 17,
            "value": "went up"
        }
    ],
    "intent": {
        "confidence": 0.8298820098064409,
        "name": "positive"
    },
    "intent_ranking": [
        {
            "confidence": 0.8298820098064409,
            "name": "positive"
        },
        {
            "confidence": 0.1701179901935592,
            "name": "negative"
        }
    ],
    "text": "Cognizant shares went up"
}
```
2. CTSH company guidence is not promising
```
{
    "entities": [
        {
            "end": 9,
            "entity": "Company",
            "extractor": "ner_crf",
            "processors": [
                "ner_synonyms",
                "ner_synonyms"
            ],
            "start": 0,
            "value": "Cognizant Technology Solutions"
        },
        {
            "end": 16,
            "entity": "feature",
            "extractor": "ner_crf",
            "start": 10,
            "value": "shares"
        },
        {
            "end": 24,
            "entity": "positive",
            "extractor": "ner_crf",
            "start": 17,
            "value": "went up"
        },
        {
            "end": 4,
            "entity": "Company",
            "extractor": "ner_crf",
            "processors": [
                "ner_synonyms"
            ],
            "start": 0,
            "value": "Cognizant Technology Solutions"
        },
        {
            "end": 38,
            "entity": "negative",
            "extractor": "ner_crf",
            "start": 25,
            "value": "not promising"
        }
    ],
    "intent": {
        "confidence": 0.8054743377038256,
        "name": "negative"
    },
    "intent_ranking": [
        {
            "confidence": 0.8054743377038256,
            "name": "negative"
        },
        {
            "confidence": 0.1945256622961744,
            "name": "positive"
        }
    ],
    "text": "CTSH company guidence is not promising"
}
```
3. CTS layoff continues.
```
{
    "entities": [
        {
            "end": 9,
            "entity": "Company",
            "extractor": "ner_crf",
            "processors": [
                "ner_synonyms",
                "ner_synonyms",
                "ner_synonyms"
            ],
            "start": 0,
            "value": "Cognizant Technology Solutions"
        },
        {
            "end": 16,
            "entity": "feature",
            "extractor": "ner_crf",
            "start": 10,
            "value": "shares"
        },
        {
            "end": 24,
            "entity": "positive",
            "extractor": "ner_crf",
            "start": 17,
            "value": "went up"
        },
        {
            "end": 4,
            "entity": "Company",
            "extractor": "ner_crf",
            "processors": [
                "ner_synonyms",
                "ner_synonyms"
            ],
            "start": 0,
            "value": "Cognizant Technology Solutions"
        },
        {
            "end": 38,
            "entity": "negative",
            "extractor": "ner_crf",
            "start": 25,
            "value": "not promising"
        },
        {
            "end": 3,
            "entity": "Company",
            "extractor": "ner_crf",
            "processors": [
                "ner_synonyms"
            ],
            "start": 0,
            "value": "CTS"
        },
        {
            "end": 10,
            "entity": "feature",
            "extractor": "ner_crf",
            "start": 4,
            "value": "layoff"
        }
    ],
    "intent": {
        "confidence": 0.8055118296377975,
        "name": "negative"
    },
    "intent_ranking": [
        {
            "confidence": 0.8055118296377975,
            "name": "negative"
        },
        {
            "confidence": 0.1944881703622027,
            "name": "positive"
        }
    ],
    "text": "CTS layoff continues."
}
```
In second and third JSON result , you can see that entities from previous two JSON's are getting appended. As you can see "CTS layoff continues" is the text which i passed and it does not contain words like "not promising","went up" and "shares" but first text and second text does.

I think for each request in parse method we have to clear the variable first which stores the response json / entity json part.

**Comment by ni1s** [comment]
I get this error too when using interpreter.parse() directly from python.

**Comment by tmbo** [comment]
@ni1s @srijiths this has been fixed in the latest release 0.8.2 (should be available on pypi), sorry for that.

**Comment by srijiths** [comment]
@tmbo Its working fine now. Thanks !
